test/uk/gov/hmrc/economiccrimelevystubs/controllers/SubscriptionControllerSpec.scala

```diff
@@ -61,4 +62,35 @@ class SubscriptionControllerSpec extends SpecBase {
     }
   }
 
+  "getSubscription" should {
+    "return 200 OK with response for individuals if reference ends with '001' " in {
+      val eclReference = "XMECL0000000001"
+
+      val result: Future[Result] = controller.getSubscription(eclReference)(fakeRequest)
+
+      status(result)        shouldBe OK
+      contentAsJson(result) shouldBe Json.toJson(GetSubscriptionData.validIndividualSubscription(eclReference))
+    }
+
+    "return 200 OK with response for organizations if reference ends with '002'" in {
+      val eclReference = "XMECL0000000002"
+
+      val result: Future[Result] = controller.getSubscription(eclReference)(fakeRequest)
+
+      status(result)        shouldBe OK
+      contentAsJson(result) shouldBe Json.toJson(GetSubscriptionData.validOrganisationSubscription(eclReference))
+    }
+
+    "return 422 UnprocessableEntity with code and message if reference ends with '422'" in {
+      val eclReference = "XMECL000000422"
+
+      val result: Future[Result] = controller.getSubscription(eclReference)(fakeRequest)
+
+      status(result)        shouldBe UNPROCESSABLE_ENTITY
+      contentAsJson(result) shouldBe Json.obj(
+        "code"   -> "006",
+        "reason" -> "There are no successfully processed forms for this customer"
+      )
+    }
```

Comment:
Can you also add a test for ServiceUnavailable and InternalServerError?

Reply:
Done :)